coda-prototype/src/ui/tableUI.js

```diff
@@ -965,11 +966,11 @@ var messageViewerManager = {
         }
     },
 
-    addNewSchemeColumn: function(scheme) {
+    addNewSchemeColumn: function(newScheme) {
 
-        if (scheme["codes"].size === 0) return; // shouldnt happen because fields are validated
+        if (newScheme["codes"].size === 0) return; // shouldnt happen because fields are validated
```

Comment:
TODO - If it shoudn't happen please log an assert failure

Reply:
I'm not sure a function should trust that it's being used correctly. I would prefer to log an error and return instead.

Reply:
Actually, now I think about it, why shouldn't a code scheme be allowed to have no codes?
In any case, this function is working fine without this test (and it makes no reference to "codes" after this point), perhaps I can delete it instead?